django_cas/views.py

```diff
@@ -16,7 +16,7 @@ def _service_url(request, redirect_to=None):
     """Generates application service URL for CAS"""
 
     protocol = ('http://', 'https://')[request.is_secure()]
-    host = get_host(request)
+    host = request.get_host()
```

Comment:
See [`get_host` in django 1.4](https://github.com/django/django/blob/stable/1.4.x/django/http/__init__.py#L783-L785).